app/screens/ChatbotScreen.tsx

```diff
@@ -0,0 +1,361 @@
+import { useNavigation } from '@react-navigation/native';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Linking, StyleSheet, View } from "react-native";
+import { TouchableHighlight } from 'react-native-gesture-handler';
+import { IconButton, Text, TextInput } from 'react-native-paper';
+import AnswerDatePicker from '../components/chatbot/AnswerDatePicker';
+import AnswerList from '../components/chatbot/AnswerList';
+import AnswerTextbox from '../components/chatbot/AnswerTextbox';
+import Loading from '../components/util/Loading';
+import Colors from '../config/Colors';
+import Routes from '../config/Routes';
+import { DeFiButton } from '../elements/Buttons';
+import { SpacerH, SpacerV } from '../elements/Spacers';
+import { H2, H3 } from '../elements/Texts';
+import withSettings from '../hocs/withSettings';
+import { ChatbotAnswer, ChatbotAPIAnswer, ChatbotAPIQuestion, ChatbotElement, ChatbotPage, ChatbotStatus } from '../models/ChatbotData';
+import { chatbotCreateAnswer, chatbotFeedQuestion, chatbotFillAnswerWithData, chatbotIsEdit, chatbotRestorePages, chatbotShouldSendAnswer, chatbotStart } from '../services/ChatbotUtils';
+import { getAuthenticationInfo, getStatus, getUpdate, nextStep, postSMSCode, requestSkip, requestSMSCode } from '../services/ChatbotService';
+import NotificationService from '../services/NotificationService';
+import { AppSettings } from '../services/SettingsService';
+import AppStyles from '../styles/AppStyles';
+
+const ChatbotScreen = ({
+  settings,
+  sessionUrl,
+  onFinish
+}: {
+  settings?: AppSettings;
+  sessionUrl: string;
+  onFinish: () => void;
+}) => {
+  const { t } = useTranslation();
+  const nav = useNavigation();
+
+  const [sessionId, setSessionId] = useState<string | undefined>();
```

Comment:
initialised useStated are always automatically `<T | undefined>()`, no need to define it here